reference/pipeline/stages/index.md

```diff
@@ -53,7 +53,8 @@ your cluster is a particular size, or add a pipeline expression. See the
 information about creating and using pipeline expressions.
 
 ### Clone Server Group
-Deploy a new Server Group that is a copy of the specified Server Group.
+Deploy the image from the original Server Group to a new Server Group. You can
```

Comment:
This reads a bit like the only thing that is by default cloned is the image, and one can choose to _also_copy the capacity. It would be more accurate to say that all the attributes of the specified server group are cloned, and one can choose to override the capacity.